CI/update_docker.sh

```diff
@@ -30,22 +29,18 @@ for ubuntu_version in ${ubuntu_versions}; do
                                   --build-arg COMPILER=${compiler} \
                  -f CI/Dockerfile_1_external_deps .
     if [ "$PUSH" = true ]; then docker push ${image_name}; fi
-    for hdf5 in $hdf5_versions; do
```

Comment:
Don't we still want/need HDF5 variations?

Reply:
we use only one everywhere.... so we might as well remove them for simplicity...
if we need more we can always add that later....

Reply:
I thought we tested the default version as well as the 1.10.4 version

Reply:
In /CI/Dockerfile we invoke this with HDF5=system

Reply:
it is an available option, but I just checked we don't use it anymore in CI

Reply:
Can we keep the infrastructure for doing this, even if we aren't using it?  It will be easier to have loops of length 1 than to recreate this if we want it again.